src/main/kotlin/com/terraformation/backend/accelerator/model/SubmissionModel.kt

```diff
@@ -0,0 +1,39 @@
+package com.terraformation.backend.accelerator.model
+
+import com.terraformation.backend.db.accelerator.DeliverableId
+import com.terraformation.backend.db.accelerator.SubmissionDocumentId
+import com.terraformation.backend.db.accelerator.SubmissionId
+import com.terraformation.backend.db.accelerator.SubmissionStatus
+import com.terraformation.backend.db.accelerator.tables.references.SUBMISSIONS
+import com.terraformation.backend.db.default_schema.ProjectId
+import org.jooq.Field
+import org.jooq.Record
+
+data class SubmissionModel<ID : SubmissionId?>(
+    val id: ID,
+    val deliverableId: DeliverableId,
+    val feedback: String? = null,
+    val internalComment: String? = null,
+    val projectId: ProjectId,
+    val submissionDocumentIds: Set<SubmissionDocumentId>,
+    val submissionStatusId: SubmissionStatus,
```

Comment:
Maybe remove the `Id` suffix from this field name since from the model's point of view, this is just an enum value rather than a numeric ID.